webtau-junit5/src/main/java/com/twosigma/webtau/junit5/Webtau.java

```diff
@@ -14,8 +14,19 @@
  * limitations under the License.
  */
 
-package com.twosigma.webtau.runner.standalone
+package com.twosigma.webtau.junit5;
 
-interface StandaloneTestResultPayload {
-    Map<String, ?> toMap()
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+
+@Target({ TYPE, ANNOTATION_TYPE })
```

Comment:
Why ANNOTATION_TYPE?

Reply:
You could have your own Extension say @IntegrationTests and annotate it wit @WebTau to inherit additional listeners. Strictly speaking we don't need to support this scenario, but that's the general idea of Junit 5 extensions (my current understanding)